subiquity/server/controllers/filesystem.py

```diff
@@ -408,7 +408,8 @@ async def v2_guided_GET(self) -> GuidedStorageResponseV2:
 
         for disk in self.get_guided_disks(with_reformatting=True):
             reformat = GuidedStorageTargetReformat(disk_id=disk.id)
```

Comment:
minor: this could be moved in the if block I think.

Reply:
Done, thanks.